src/components/hero/campaignHeroSubText.tsx

```diff
@@ -46,10 +48,10 @@ const CampaignHeroSubText = () => {
               color: "var(--nypl-colors-ui-link-tertiary) !important",
             },
           }}
-          href="https://digitalcollections.nypl.org/items/510d47e0-cb17-a3d9-e040-e00a18064a99"
+          href={featuredItem.href}
         >
           {" "}
-          Momoyogusa = Flowers of a Hundred Generations.{" "}
+          {featuredItem.title.$}{" "}
```

Comment:
@EdwinGuzman for longer titles, I was wondering how we could break them up to look more like the titles in the figma file https://www.figma.com/file/NpjG47HqZG9GknfijGqJri/DC-Facelift-Homepage?type=design&node-id=3966-5868&mode=design&t=KLTjKriR6HBxc8Jz-0

Reply:
Try updating to `display: inline`. It's currently set to `display: inline-block`. A few examples:

<img width="583" alt="Screen Shot 2023-11-09 at 12 14 21 PM" src="https://github.com/NYPL/digital-collections/assets/1280564/7b38155f-c071-4b7f-b225-e1df0a3f3bea">
<img width="644" alt="Screen Shot 2023-11-09 at 12 13 36 PM" src="https://github.com/NYPL/digital-collections/assets/1280564/6b62ef88-8a52-43c7-a0b9-846ffaefeda9">
(on mobile)
<img width="473" alt="Screen Shot 2023-11-09 at 12 14 30 PM" src="https://github.com/NYPL/digital-collections/assets/1280564/5b06f743-a65f-45a0-8906-3b13885f0703">

In the Figma file, the mobile view has it at a max two-lines long. @7emansell recently did text truncation that _might_ work in this case and you'd have to add it to the `Text` parent component. Charka docs: https://v1.chakra-ui.com/docs/components/typography/text#truncate-text